qiskit/circuit/quantumcircuit.py

```diff
@@ -4178,3 +4213,19 @@ def _insert_composite_gate_definition_qasm(
 
     string_temp = string_temp.replace(extension_lib, f"{extension_lib}{gate_definition_string}")
     return string_temp
+
+
+def _warn_on_broadcasting_controlled_gate(qargs, name):
+    # check for multiple qubits
+    if any(isinstance(qarg, (tuple, list, np.ndarray)) and len(qarg) > 1 for qarg in qargs):
+        warnings.warn(
+            "Creating multiple single-controlled gates with a single call, "
+            "as QuantumCircuit.cx([0, 1], 2) is deprecated as of Qiskit "
+            "Terra 0.19.0 and will be unsupported no sooner than 3 months after its "
+            "release date. In future, this will construct a multi-controlled gate, where the "
+            "first argument determines the control qubits and the second the target qubit(s). "
+            f"To construct multiple gates, you can call the {name} method multiple times with a "
+            "single control and target.",
+            FutureWarning,
+            stacklevel=3,
```

Comment:
I accidentally changed this to `stacklevel=3` when I pushed a commit fixing the docs build (I was debugging with it, and only meant to comment, not push that bit), but I think it's correct anyway - the stack is:
1. `warnings.warn`
2. `_warn_on_broadcasting_controlled_gate`
3. the bad call

Feel free to revert if I've got it wrong.